src/components/Goods/Goods.jsx

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import { Good } from '../Good';
+import './Goods.scss';
+
+export class Goods extends React.Component {
+  state = {
+    selected: ' - ',
+    clearMode: false,
+  }
+
+  clearAll = () => {
+    this.setState({
+      clearMode: true,
+    });
+  }
+
+  selected = ({ target }, item) => {
+    this.setState({
+      selected: ` ${item} `,
+      activeItem: target,
+      clearMode: false,
+    });
+  }
+
+  render() {
+    const { goods } = this.props;
+    const { selected, activeItem, clearMode } = this.state;
+
+    return (
+      <div className="goods">
+        <div className="goods__header-wrapper">
+          <h1>
+            Selected good:
+            {selected}
+          </h1>
+          <button
+            title="clear selection"
+            type="button"
+            className="goods__clear"
+            onClick={() => {
+              this.setState({
+                selected: ' - ',
+              });
+              this.clearAll();
```

Comment:
You can write `this.setState({selected: ' - '})` directly in your `clearAll` function